docs/r2/tensorboard_in_notebooks.ipynb

```diff
@@ -77,7 +77,14 @@
       },
       "cell_type": "markdown",
       "source": [
-        "TensorBoard can be used directly within notebook experiences such as [Colab](https://colab.research.google.com/) and [Jupyter](https://jupyter.org/). This can be helpful for sharing results, integrating TensorBoard into existing workflows, and using TensorBoard without installing anything locally."
+        "TensorBoard can be used directly within notebook experiences such as [Colab](https://colab.research.google.com/) and [Jupyter](https://jupyter.org/). This can be helpful for sharing results, integrating TensorBoard into existing workflows, and using TensorBoard without installing anything locally.\n",
+        "\n",
+        "**For Jupyter users:** If you’ve installed Jupyter and TensorBoard into\n",
+        "the same virtualenv, then you should be good to go. If you’re using a\n",
+        "more complicated setup, like a global Jupyter installation with a custom\n",
+        "kernelspec, then you must ensure that the `tensorboard` binary is on\n",
+        "your `PATH` inside the Jupyter notebook context by prepending your\n",
+        "virtualenv’s `bin` directory to `os.environ[\"PATH\"]`."
```

Comment:
Maybe it's just me, but is the reference to the "virtualenv's `bin`" directory is a bit confusing given that this "more complicated setup" doesn't necessarily use virtualenvs?

Reply:
May I suggest rephrasing the sentence like this?


> In a more complex setup, with a global Jupyter installation and kernels for different conda/virtualenv environments, you must ensure that the `tensorboard` binary is on your `PATH` *inside the Jupyter notebook context*. One solution is modifying the `kernel_spec` to prepend the PATH with the environment's `bin` dir before launching the kernel as [described here](https://github.com/ipython/ipykernel/issues/395#issuecomment-479787997).

Reply:
Done; thanks for the suggestion. Please edit the linked issue comment to
change `$*` to `"$@"` (note quotes) in the shell script.